.github/workflows/ci-complete.yml

```diff
@@ -67,7 +62,7 @@ jobs:
       - name: Download build scan archive
         id: download-build-scan
         uses: actions/download-artifact@v4
-        continue-on-error: true
+        continue-on-error: true  # Don't want this step to fail the overall workflow
```

Comment:
This prevents the CI Complete workflow from failing if the PR didn't produce a build scan